ebssurrogate/USAGE.md

```diff
@@ -0,0 +1,71 @@
+## Ext4 amd64 AMI creation
+
+`packer build -var "aws_access_key=$AWS_ACCESS_KEY_ID" -var "aws_secret_key=$AWS_SECRET_ACCESS_KEY" -var "region=$AWS_REGION" \
```

Comment:
Is there a reason we would want to retain all these different combinations? I'd be in favour of removing configurability here, beyond arm vs x86.

Reply:
There are two different variants: 1. using `fast-commit` with ext4 and  2. ZFS support for amd64. 
1. Removed this option now, because we are not going to use `fast-commit`  without proper testing for this feature. 
2. Removed this too, ZFS supported only for amd64, I had issues with zfs+arm ami. This can be added later.